jupyter_server/gateway/managers.py

```diff
@@ -43,13 +49,33 @@ def _default_kernel_manager_class(self):
     def _default_shared_context(self):
         return False  # no need to share zmq contexts
 
+    event_schema_id = os.path.join(JUPYTER_SERVER_EVENTS_URI, "kernel_actions", "v1")
```

Comment:
`os.path.join` is not right here. This is a URI, not a file path. Using `os.path.join` here would break things on Windows by inserting a backwards slash.